src/main/kotlin/fi/aalto/cs/apluscourses/utils/parser/FeedbackParser.kt

```diff
@@ -0,0 +1,9 @@
+package fi.aalto.cs.apluscourses.utils.parser
+
+open class FeedbackParser {
```

Comment:
I might add them back in the next version to show feedback for optional assignments. The feedback parsing was used before to show if all the tests in an optional assignment passed. It might not be needed anymore though, because now optional assignments give 1 point instead of 0, so the user still should get feedback if all tests pass.

Another use for feedback parsing would be to show that even if the user gets full points, there are still failing optional tests